ci/build-docker.sh

```diff
@@ -22,10 +22,23 @@ set -eu
 mkdir -p gccpp/build-output
 cd gccpp/build-output
 
-cmake -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" ${CMAKE_FLAGS:-} ..
+CMAKE_COMMAND="cmake"
+if [ "${SCAN_BUILD}" = "yes" ]; then
+    CMAKE_COMMAND="scan-build cmake"
```

Comment:
It's 2-space indent elsewhere in the file (below).

Reply:
Done.